test/Java8andUp/src/org/openj9/test/vmcheck/TestLoadingClassesFromJarfile.java

```diff
@@ -100,6 +103,14 @@ private void loadClassesFromJarfile(String jarfilePath) {
 					String jarEntryName = entry.getName();
 					if (filterFileName(jarEntryName)) {
```

Comment:
The `private boolean filterFileName(String jarEntryName) {` method provides a mechanism to skip AWT-related classes.  Can that mechanism be extended to cover the new cases of these compile stubs?